packages/viewer/src/services.ts

```diff
@@ -105,7 +96,7 @@ export async function mathml2accessible(mml: string, lang: string, url: string,
     ignoreStyles: "true",
   };
 
-  const response = callService(params, "service", MethodType.Post, url, extension);
+  const response = await callService(params, "service", MethodType.Post, url, extension);
```

Comment:
This is related to the previous comment.  
If we don't handle these promises, any resulting error messages from failed promises will come from the parent caller, resulting in somewhat confusing messages, which could make debugging very tricky.